dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/RenormalizationProcessorTest.java

```diff
@@ -87,31 +107,133 @@ private int computeEnd(int position, String reference, String alternate) {
     }
 
     @Test
     public void ambiguousDeletions() throws Exception {
-        assertMatchesExpected(3, "CG", "", 2, 3, "GC", "");
-        assertMatchesExpected(5, "CG", "", 4, 5, "GC", "");
-        assertMatchesExpected(6, "C", "", 5, 5, "C", "");
+        assertMatchesExpected(3, "CG", "", 2, 3, "GC", ""); // 2:GCG>G
+        assertMatchesExpected(5, "CG", "", 4, 5, "GC", ""); // 4:GCG>G
+        assertMatchesExpected(6, "C", "", 5, 5, "C", "");   // 5:CC>C
     }
 
     private void assertMatchesExpected(int position, String reference, String alternate, int expectedStart,
                                        int expectedEnd, String expectedReference,
                                        String expectedAlternate) throws Exception {
         int endPosition = computeEnd(position, reference, alternate);
         Variant variant = new Variant("22", position, endPosition, reference, alternate);
-        Variant renormalized = renormalizer.process(variant);
+        IVariant renormalized = renormalizer.process(variant);
         assertNotNull(renormalized);
         assertEquals(expectedStart, renormalized.getStart());
         assertEquals(expectedEnd, renormalized.getEnd());
         assertEquals(expectedReference, renormalized.getReference());
         assertEquals(expectedAlternate, renormalized.getAlternate());
     }
-}
\ No newline at end of file
+
+    @Test
+    public void variantIsCopiedCompletely() throws Exception {
+        //given
+        int start = 3;
+        String reference = "";
+        String alternate = "CG";
+        String renormalizedAlternate = "GC";
+        int endPosition = computeEnd(start, reference, alternate);
+        Variant variant = new Variant("22", start, endPosition, reference, alternate);
+        String mainId = "rsId";
+        HashSet<String> dbsnpIds = new HashSet<>(Arrays.asList(mainId, "ssId_1", "ssId_2"));
+        variant.setDbsnpIds(dbsnpIds);
+        variant.setMainId(mainId);
+        Map<String, VariantStatistics> cohortStats = new HashMap<>();
+        cohortStats.put("ALL", new VariantStatistics(reference, alternate, VariantType.INDEL, (float) 0.2, (float) 0.3,
+                                                     alternate, "0/1", 1, 2, 3, (float) 0.4, (float) 0.5,
+                                                     (float) 0.6, (float) 0.7));
+
+        Map<String, String> attributes = new HashMap<>();
+        attributes.put(DBSNP_BUILD_KEY, "test_release");
+
+        List<Map<String, String>> samplesData = new ArrayList<>();
+        samplesData.add(Collections.singletonMap("GT", "0.1"));
+
+        VariantSourceEntry variantSourceEntry = new VariantSourceEntry("fileId", "studyId", new String[]{"T"}, "GT",
+                                                                       cohortStats, attributes, samplesData);
+        variant.addSourceEntry(variantSourceEntry);
+
+        //when
+        IVariant renormalized = renormalizer.process(variant);
```

Comment:
`assertNotNull(renormalized)` after this for more clear detection of where an issue could be; otherwise we would just get NPE in the next assert.